trackintel/model/util.py

```diff
@@ -31,7 +33,10 @@ def get_speed_positionfixes(positionfixes):
     g = pfs.geometry
     # get distance and time difference
     if is_planar_crs:
-        dist = g.distance(g.shift(1)).to_numpy()
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", message="CRS not set for some of the concatenation inputs.*")
```

Comment:
Why are the warnings filtered here? Isn't it a problem if the crs is not set for the positionfixes?

Reply:
Ah, is this problem due to the g.shift? After it's shifted, there is no crs defined anymore?

Reply:
You might be right. But I am not sure why a warning is raised :), see below (or above) comments.

Reply:
Seems like this was a bug in the past but is fixed now: https://github.com/geopandas/geopandas/issues/1648

Reply:
The warning persists in our test environment that uses geopandas 0.12.2